src/controllers/postControllers.js

```diff
@@ -0,0 +1,27 @@
+import {postRepository} from '../repositories/index.js'
+ 
+
+async function timeline(req, res){
+    try{
+        const posts = await postRepository.getPost();
+        res.status(200).send(posts);
+    }
+    catch{
+        res.sendStatus(500);
+    }
+}
+
+async function sendPost(req, res){
+    const {content, postLink} = req.body;
```

Comment:
It is better to get the information you need from the previous middleware, that way you make sure that the pre-requesites are met in case you forget a return on the middleware.